modules/client-configuration/pages/clients-microos.adoc

```diff
@@ -0,0 +1,144 @@
+[[clients-microos]]
+= Registering {opensuse}{nbsp}{microos} Clients
+
+This section contains information about registering clients running these {opensuse}{nbsp}{microos} operating systems:
+
+* {opensuse}{nbsp}{microos}
+
+
+
+[WARNING]
+====
+This section is provided as a technology preview for testing purposes.
+Do not use this feature on production systems.
+====
+
+
+
+== Add Software Channels
+
+Before you register {opensuse}{nbsp}{microos} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
+
+The channels you need for this procedure are:
+
+[[microos-channels-cli]]
```

Comment:
`opensusemicroos-channels-cli`

Reply:
it really doesn't matter what we put in the labels.